src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.SslConnectionOptions.cs

```diff
@@ -42,14 +42,20 @@ public partial class QuicConnection
         /// </summary>
         private readonly RemoteCertificateValidationCallback? _validationCallback;
 
-        public SslConnectionOptions(QuicConnection connection, bool isClient, string? targetHost, bool certificateRequired, X509RevocationMode revocationMode, RemoteCertificateValidationCallback? validationCallback)
+        /// <summary>
+        /// Configured via <see cref="SslServerAuthenticationOptions.CertificateChainPolicy"/> or <see cref="SslClientAuthenticationOptions.CertificateChainPolicy"/>.
+        /// </summary>
+        private readonly X509ChainPolicy? _chainPolicy;
+
+        public SslConnectionOptions(QuicConnection connection, bool isClient, string? targetHost, bool certificateRequired, X509RevocationMode revocationMode, RemoteCertificateValidationCallback? validationCallback, X509ChainPolicy? chainPolicy)
```

Comment:
nit: as this gets longer and longer it may be nice to split it to multiple lines.